public/js/AdminCtrl.js

```diff
@@ -45,8 +45,8 @@ polr.controller('AdminCtrl', function($scope, $compile) {
                 "ajax": BASE_API_PATH + 'admin/get_admin_links',
 
                 "columns": [
-                    {className: 'wrap-text', data: 'short_url', name: 'short_url'},
-                    {className: 'wrap-text', data: 'long_url', name: 'long_url'},
+                    {data: 'short_url', name: 'short_url'},
```

Comment:
Shouldn't we still wrap-text even if we're trimming the link? Otherwise, the link will bleed into the neighboring fields.